packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -296,7 +298,8 @@ export const DEFAULT_THEME: DocsThemeConfig = {
   toc: {
     component: TOC,
     float: true,
-    title: 'On This Page'
+    title: 'On This Page',
+    extraContent: <React.Fragment />
```

Comment:
```suggestion
    title: 'On This Page'
```
this is redundant

Reply:
Running into an error too for the docs theme

Reply:
found why we didn't get type errors in this case:

the problem was in mismatching `z.custom` validation function and `ReactNode` type, `ReactNode` accept a value to be `null` or `undefined` but our custom `isReactNode` didn't allowed it, so I adjusted the function by checking `value == null`

I pushed a fix in this branch